packages/blade/src/components/BaseHeaderFooter/BaseFooter.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Divider } from './Divider';
+import BaseBox from '~components/Box/BaseBox';
+import { assignWithoutSideEffects } from '~utils';
+
+export type BaseFooterProps = {
+  children: React.ReactNode;
+  hideDivider?: boolean;
```

Comment:
Because by default divider is shown.

if we call it showDivider, then to hide the divider people will have to do <BottomSheetHeader showDivider={false} /> instead of <BottomSheetHeader hideDivider />

Reply:
Right, if we've to hide the divider we can pass `showDivider={false}`, I think we've been following `show*` terminology for more clarity at other places